src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1539,7 +1539,6 @@ internal enum ErrorCode
         ERR_OutAttrOnInParam = 8355,
         #endregion diagnostics introduced for `ref readonly`, `ref ternary` and `ref-like` features in C# 7.2
 
-        #region diagnostics introduced for C# 7.3
```

Comment:
Was this change intentional?

Reply:
Yes, this is intentional.
I'm not sure why this comment landed here. But there is no LanguageVersion for C# 7.3 in master ([see](https://github.com/dotnet/roslyn/blob/master/src/Compilers/CSharp/Portable/LanguageVersion.cs#L121)).

Reply:
Work for C# 7.3 is happening in the `features/compiler` branch (which may align with15.7).

Reply:
@gafter I see that that comment was added in one of your PRs (https://github.com/dotnet/roslyn/pull/23265/files#diff-3ff29d8326aea505287a1f262172c8a3). FYI, I'm removing it.